src/launch/launch.scss

```diff
@@ -22,9 +22,15 @@
   .subtext {
     font-family: Inter;
     font-weight: normal;
-    font-size: 0.85em;
+    font-size: 18px;
     color: $Neutral03;
   }
+
+  .banner > .subtext {
+    font-weight: bold;
+    text-align: center;
+    width: 615px;
```

Comment:
did you test this at various widths?  Would probably be better as % or [n]vw

Reply:
Yes it works at various width of the viewport but I can also make it em, % or vw also

Reply:
Okay so I tried this with various viewport width and it is fine, should I leave as is or change to some relative width value?